mitmproxy/optmanager.py

```diff
@@ -20,6 +20,27 @@
 unset = object()
 
 
+class Path(str):
```

Comment:
Would it make sense to use [`pathlib.Path`](https://docs.python.org/3/library/pathlib.html) instead?